cachelib/allocator/Refcount.h

```diff
@@ -246,65 +264,131 @@ class FOLLY_PACK_ATTR RefcountWithFlags {
   }
 
   /**
-   * The following four functions are used to track whether or not
-   * an item is currently in the process of being moved. This happens during a
-   * slab rebalance or resize operation or during eviction.
+   * The following two functions corresond to whether or not an item is
+   * currently in the process of being evicted. When item is marked exclsuive
```

Comment:
What's your future plan for `markExclusive()`? Is it also used in a case other than eviction?
If not, I think we can rename this called markEviction() where markEviction means marking exclusive bit with access ref==0, isLinked and isAccessible.